tools/test-proxy/scripts/transition-scripts/README.md

```diff
@@ -8,16 +8,46 @@ The script `generate-assets-json.ps1` will execute the initial migration of your
+
+```bash
+wget https://raw.githubusercontent.com/Azure/azure-sdk-tools/main/tools/test-proxy/scripts/transition-scripts/generate-assets-json.ps1
+```
+
 ## Setup
 
-Before running the script, understand that **only services that have migrated to use the `test-proxy` as their record/playback solution can store recordings into the external assets repository.** The CLI command commands are invoked from the `test-proxy` itself, so if it is NOT being used for record/playback, the work to transition to the `test-proxy` must be done before recordings can be moved.
+Before running the script, understand that **only services that have migrated to use the `test-proxy` as their record/playback solution can store recordings into the external assets repository.** The test-proxy itself contains the code for `restoring`/`push`ing recordings, so if it is NOT being used for record/playback, that work must be completed before recordings can be moved.
 
-Running the script requires the following:
+Running the script requires these base requirements.
 
 - [x] The targeted library is already migrated to use the test-proxy.
 - [x] Git version `>2.25.0` needs to be on the machine and in the path. Git is used by the script and test-proxy.
-- [x] Test-proxy needs to be on the machine and in the path. Instructions for that are [here](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md#installation). The script uses test-proxy's CLI commands.
 - [x] [Powershell Core](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2) at least version 7.
+- [x] Ensure global git config settings for `user.name` and `user.email` are updated. [Reference](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup)
+  - Override with environment variables `GIT_COMMIT_EMAIL` and `GIT_COMMIT_OWNER`. If either of these are set, they will override the default values pulled from `git config --global`.
+
+Once the above requirements are met, developers are welcome to choose one of the following paths.
+
+### `test-proxy` dotnet tool installed and called directly
+
+Provide `TestProxyExe` argument of `test-proxy` or leave it **blank**. This is the default use-case of this transition script.
+
+- [x] Test-proxy needs to be on the machine and in the path. Instructions for that are [here](https://github.com/Azure/azure-sdk-tools/blob/main/tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md#installation).
+
+The newly installed test-proxy tool will be used during the recording migration portion of this script.
+
+### `docker` or `podman` invocation
+
+To utilize this methodology, the user must set input argument `TestProxyExe` to `docker` or `podman`.
+
+Other requirements:
+
+- [x] Install [docker](https://docs.docker.com/engine/install/) or [podman](https://podman.io/getting-started/installation.html)
+- [x] Set the environment variable `GIT_TOKEN` a valid token representing YOUR user
```

Comment:
This is only a requirement now if you are using https auth. Don't know if you want to specify the ssh exception here as well, or just let people discover it on their own if they aren't using https.

Reply:
Hm, I guess it's kind of more annoying because you'd have to inject it into the test proxy command somehow...

Reply:
Nvm, see below comment, I think we can handle this (happy to do an update myself).